packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -359,6 +370,23 @@ export class MuteState {
     }
   }
 
+  public shouldIgnoreRemoteMuteUpdate(remoteMute: boolean, isModifiedBySelf: boolean) {
+    console.log(
+      `marcin: shouldIgnoreRemoteMuteUpdate: flag=${this.state.isRemoteUnmutePendingLocusDtoUpdate} remoteMute=${remoteMute}, isModifiedBySelf=${isModifiedBySelf}`
+    );
+    if (this.state.isRemoteUnmutePendingLocusDtoUpdate && isModifiedBySelf && !remoteMute) {
```

Comment:
We only ever send remote unmute requests (not mute), so here I'm only checking for `!remoteMute`, but the method that sends out these requests (`sendRemoteMuteRequestToServer()`) is written in a way that suggests that it could do either mute or unmute, so I think I'll rename `sendRemoteMuteRequestToServer()` to `sendRemoteUnmuteRequestToServer()` and hardcode `remoteMute = false` in it to make it more clear that we only do remote unmute, then this will be more in-line with the hardcoded assumption here in `shouldIgnoreRemoteMuteUpdate()`